src/main/java/Tasks/Event.java

```diff
@@ -0,0 +1,21 @@
+package tasks;
+
+import exceptions.NoTaskDescriptionException;
+
+public class Event extends Task{
+    String startTime;
+    String endTime;
+
+    protected Event(String name, String startTime, String endTime) throws NoTaskDescriptionException{
+        super(name, "Event");
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    @Override
+    public String toString() {
+        return "[E] " + this.TasktoString()
+            + "( from: " + this.startTime 
+            + ") ( to: " + this.endTime + ")";
```

Comment:
Perhaps increase the indentation of wrapped lines to 8 spaces to match the coding standard?

```suggestion
        return "[E] " + this.TasktoString()
                + "( from: " + this.startTime 
                + ") ( to: " + this.endTime + ")";
```